### ISSUE TITLE
Entity id Collisions when multiple magicmirror instances

### ISSUE THREAD

**Issue by sisimomo** [sindrebroch/ha-magicmirror#15]
## Version of the custom_component
1.3.0

## Describe the bug
I have 2 smart mirrors in my house. When I add the second mirror I have a collision between entity ids. That cause some strange behavior like 1 entity for both mirror:

![image](https://user-images.githubusercontent.com/27745204/190561376-f1e1e02b-9b92-4022-8ca8-a913795daed5.png)

Collisions obviously only happen when both mirrors have the same module as well as the mirror's basic controls.

**Comment by sindrebroch** [comment]
Right. I haven't really tested adding more than one mirror, so this makes sense. Will look into this, and also probably adapt to the new naming standard for HA.

Thanks for reporting.
